### ISSUE TITLE
Charter: ambiguous use of the term "embedded"

### ISSUE THREAD

**Issue by cookiecrook** [w3c/wcag#2761]
[Proposed Charter](https://www.w3.org/2022/10/proposed-ag-charter) includes:

> WCAG 3 will not include normative guidance for stand-alone user agents or authoring tools, however, it may include guidelines for _**embedded**_ user agents and authoring tools such as an online video player built with JavaScript and HTML, or online blogging service.

"embedded" is possibly misused here, or ambiguous at least. Suggest replacing "embedded user agents and authoring tools" with "web-based user interface display technologies" or something else similarly specific.

**Comment by bruce-usab** [comment]
Since "stand-alone user agents" is clear enough, I agree that embedded must be a reference to text editing widgets not unlike the commenting system used for GitHub issues!

I don't think the context is necessarily scoped to only user-generated content from the public.  A organization's CMS would be covered by WCAG3, and would be in-scope for conformance.

**Comment by bruce-usab** [comment]
I agree with trying to avoid "embedded".  I am not so keen on substituting six words for one.   I think "display" might be too limited.  Can we just use "components"?  Also, adding a period, and my nit pick about "guidelines" versus capital-G-Guidelines or lower-case-g-guidance .

> WCAG3 will not include normative guidance for stand-alone user agents or authoring tools.  WCAG3 may include Guidelines for web-based components and authoring tools, such as an online video player built with JavaScript and HTML, or an online blogging service.

**Comment by cookiecrook** [comment]
> web-based components and authoring tools

That phrasing works for me. Thanks.

**Comment by awkawk** [comment]
I'm not so sure. The idea of having authoring rules is more complex than many may think. In the Section 508 refresh we struggled with authoring tool language because in some cases an authoring tool is a single tool and in many cases it is a sequence of tools (e.g., a tool to create the captioning, a different tool that creates video, a different tool that adds both to a page). 

So if a standalone tool is used as part of that process then that tool won't need to meet the requirements specified in WCAG 3. Are we ok with specifying authoring requirements that apply to web-based tools but not those built as OS-native tools? Chances are those requirements will wind up applying to the standalone tools anyway as rule-making efforts in the EU/US/Canada/etc may apply the web-based rules to other software, so it would be better if we could build the rules with the full scope in mind.
